spec/grape/dsl/middleware_spec.rb

```diff
@@ -6,24 +6,44 @@ module MiddlewareSpec
       class Dummy
         include Grape::DSL::Middleware
       end
+
+      class MyMiddleware; end
+      class MyOtherMiddleware; end
```

Comment:
I know `Dummy` is declared above the same way, but these are global scope so we should use `let(:dummy) { Class.new(...) }` instead, if you don't mind changing those.